### PR TITLE
WFLY-5227 - move security-manager to wildfly-core

### PR DESCRIPTION
**Pull request by luck3y**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6131,7 +6124,7 @@
                 <version>${version.org.wildfly.arquillian}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>org.wildfly.security</groupId>
+                        <groupId>org.wildfly.core</groupId>
```
Comment (bstansberry) [inline pom.xml]: This is a different artifact -- it's the actual security manager, not the subsystem.

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (luck3y) [inline pom.xml]: Fixed, below, I'll squash the additional ones at the end.

**Comment by luck3y** [comment]
CI Run: https://ci.wildfly.org/viewLog.html?buildId=36006&tab=buildResultsDiv&buildTypeId=WF_WildFlyCoreIntegrationExperiments

**Comment by luck3y** [comment]
Hopefully successful run in CI, due to some confusion of -SNAPSHOT vs not  :(

https://ci.wildfly.org/viewLog.html?buildId=36033&tab=buildResultsDiv&buildTypeId=WF_WildFlyCoreIntegrationExperiments

**Comment by luck3y** [comment]
@bstansberry This is all set now, -SNAPSHOT removed and CI passed.

**Comment by bstansberry** [comment]
retest this please

**Comment by bstansberry** [comment]
retest this please

**Review by bstansberry — APPROVED** [review]

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Pull request merged**